CHANGELOG.md

```diff
@@ -1,7 +1,6 @@
 ## 1.0.14
 
 * Added trigger `Lead Status Change`
-* Added ability to pass trigger field to `Updated Lead`, `Updated Contact`, `Updated Deal`. User will be able to select field which value will be validated against trigger - Zap won't trigger if value doesnt' change
```

Comment:
Did we release `1.0.14`? If yes, maybe we should release 1.0.15 with the new entry in the changelog about removing this feature.

Reply:
Fortunately, we didn't ;)